PackageIndexer/Program.cs

```diff
@@ -0,0 +1,293 @@
+        // Generate a CSV file from each dictionary
+
+        Dictionary<string, IList<CsvEntry>> csvDictionary = [];        
+        Dictionary<string, int> packageCounter = []; // Used for "pac" number in CSV file.
+        foreach (string moniker in s_tfmToOpsMoniker.Values)
+        {
+            csvDictionary.Add(moniker, []);
+            packageCounter.Add(moniker, 1);
+        }
+
+        // Get all XML files (ignores disabled indexes).
+        IEnumerable<string> packageIndexFiles = Directory.EnumerateFiles(indexPackagesPath, "*.xml");
+        foreach (string packageIndexFile in packageIndexFiles)
+        {
+            // Read XML file to get each listed framework.
+            PackageEntry packageEntry = XmlEntryFormat.ReadPackageEntry(packageIndexFile);
+
+            Console.WriteLine($"Creating CSV entries for package {packageEntry.Name}.");
+
+            // Add to each applicable CSV file.
+            foreach (FrameworkEntry frameworkEntry in packageEntry.FrameworkEntries)
+            {
+                string framework = frameworkEntry.FrameworkName;
+                string opsMoniker;
+                switch (framework)
+                {
+                    case "net462":
+                        opsMoniker = s_tfmToOpsMoniker[framework];
+                        AddCsvEntryToDict(opsMoniker, csvDictionary, packageCounter, packageEntry, frameworkEntry);
+                        framework = "net47";
+                        goto case "net47";
+                    case "net47":
+                        opsMoniker = s_tfmToOpsMoniker[framework];
+                        AddCsvEntryToDict(opsMoniker, csvDictionary, packageCounter, packageEntry, frameworkEntry);
+                        framework = "net471";
+                        goto case "net471";
+                    case "net471":
+                        opsMoniker = s_tfmToOpsMoniker[framework];
+                        AddCsvEntryToDict(opsMoniker, csvDictionary, packageCounter, packageEntry, frameworkEntry);
+                        framework = "net472";
+                        goto case "net472";
+                    case "net472":
+                        opsMoniker = s_tfmToOpsMoniker[framework];
+                        AddCsvEntryToDict(opsMoniker, csvDictionary, packageCounter, packageEntry, frameworkEntry);
+                        framework = "net48";
+                        goto case "net48";
+                    case "net48":
+                        opsMoniker = s_tfmToOpsMoniker[framework];
+                        AddCsvEntryToDict(opsMoniker, csvDictionary, packageCounter, packageEntry, frameworkEntry);
+                        framework = "net481";
+                        goto case "net481";
+                    case "net481":
+                    case "net6.0":
+                    case "net7.0":
+                    case "net8.0":
+                    case "net9.0":
+                    case "netstandard2.0":
```

Comment:
If a package targets .NET Standard 2.0 but not 2.1, should I still add an entry for 2.1? That is, should the APIs show up in docs as being applicable to .NET Standard 2.1? I wasn't sure, because some packages like [this one](https://www.nuget.org/packages/Microsoft.Extensions.DependencyInjection) explicitly target both 2.0 and 2.1, but some only target 2.0, like [this one](https://www.nuget.org/packages/Microsoft.Extensions.Primitives).